runtime/port/sysvipc/j9sharedhelper.c

```diff
@@ -139,7 +139,7 @@ createDirectory(struct J9PortLibrary *portLibrary, char *pathname, uintptr_t per
 		
 	current = strchr(tempPath+1, DIR_SEPARATOR); /* skip the first '/' */
 
-	if (J9SH_DIRPERM_ABSENT == permission) {
+	if (J9SH_DIRPERM_ABSENT == permission || J9SH_DIRPERM_ABSENT_GROUPACCESS == permission) {
```

Comment:
formatting - please enclose each condition in its own parenthesis so that readers don't have to think about order of operations.

Reply:
Fixed